packages/runtime/package.json

```diff
@@ -23,41 +14,29 @@
     "John Otander <johnotander@gmail.com> (http://johnotander.com)",
     "Tim Neutkens <tim@zeit.co>",
     "Matija Marohnić <matija.marohnic@gmail.com>",
-    "Titus Wormer <tituswormer@gmail.com> (https://wooorm.com)"
+    "Titus Wormer <tituswormer@gmail.com> (https://wooorm.com)",
+    "JounQin <admin@1stg.me> (https://www.1stg.me)"
   ],
-  "main": "dist/index.js",
-  "module": "dist/index.es.js",
-  "scripts": {
-    "build": "microbundle --format es,cjs --external react,buble-jsx-only,@mdx-js/mdx,@mdx-js/react --jsx React.createElement --no-sourcemap",
-    "clean": "rimraf dist",
-    "prepublish": "yarn clean && yarn build",
-    "test": "jest"
-  },
+  "license": "MIT",
+  "main": "dist/cjs.js",
```

Comment:
is this change intended?
Would a build update be needed to support this?
see https://github.com/mdx-js/mdx/issues/876

Reply:
@ChristianMurphy Thx for point out, I'm working on it.